### PR TITLE
added dead alive agent for teams

### PR DESCRIPTION
**Pull request by weseldridge**
This didn't even come to mind when working on this the first time.

I am not too happy with the reordering of the agent's array to display this. If there is a better way to do it please let me know.

### CODE REVIEW

**Comment by weseldridge** [comment]
I am going to try to add "vs" leave open for now.

File: pommerman/graphics.py [hunk 1]
```diff
@@ -372,9 +372,19 @@ def render_dead_alive(self):
...
         dead.height = image_size
         sprites = []
-
-        for agent in self._agents:
-            x = self.board_right(x_offset=-3) - (4 - agent.agent_id) * (
+        
+        if self._game_type is constants.GameType.FFA:
+            agents = self._agents
+        else:
+            agents = [
```
Comment (suphoff) [inline pommerman/graphics.py]: Looks good as is - but as you asked for a suggestion:
```python
agents = [self._agents[i] for i in [0,2,1,3]]
```

Comment (cinjon) [reply pommerman/graphics.py]: 👍

File: pommerman/graphics.py [hunk 2]
```diff
@@ -372,9 +372,19 @@ def render_dead_alive(self):
...
+                self._agents[0],
+                self._agents[2],
+                self._agents[1],
+                self._agents[3]
+            ]
+
+        for index, agent in enumerate(agents):
+            x = self.board_right(x_offset=-3) - (4 - index) * (
```
Comment (cinjon) [inline pommerman/graphics.py]: A comment right here saying why you are doing this funky computation would be helpful.

**Comment by cinjon** [comment]
thanks! lgtm.

**Pull request merged**